src/app/commands/cmd_new_brush.cpp

```diff
@@ -64,6 +64,10 @@ bool NewBrushCommand::onEnabled(Context* context)
 
 void NewBrushCommand::onExecute(Context* context)
 {
+  ASSERT(current_editor);
+  if (!current_editor)
+    return;
+
```

Comment:
Whatever , I fixed the commit message.
![_d l m f g9u7672c_kkof](https://cloud.githubusercontent.com/assets/5674588/7455489/4a7de858-f2af-11e4-8ce6-8092008bf467.gif)